.travis.yml

```diff
@@ -12,9 +12,9 @@ before_install:
       echo "Skipping CI; no changed formulae found in $TRAVIS_BUILD_DIR";
       travis_terminate 0;
     else
-      echo "Changed formulae are ${CHANGED_FORMULAE}";
-    fi
+      echo "Changed formulae are ${CHANGED_FORMULAE}"; fi
```

Comment:
What's the point of this?

Reply:
if you mean the linebreak, that's by accident